packages/demos/src/webpack-base-config.js

```diff
@@ -43,6 +43,7 @@ const webpackBaseConfig = {
   },
   plugins: [
     new webpack.DefinePlugin({
+      'process.env.BLUEPRINT_NAMESPACE': '""',
```

Comment:
Why is the ENV var new?

Reply:
[Webpack 5 does no longer include a polyfill for `process`](https://webpack.js.org/migrate/5/#run-a-single-build-and-follow-advises). This env variable was/is accessed by blueprint (and they fall back to a default value if it is falsy).